test/model/src/include/model/util.h

```diff
@@ -304,6 +317,19 @@ class config_map {
         return v;
     }
 
+    /*
+     * config_map::keys --
+     *     Get the collection of keys.
+     */
+    inline std::vector<std::string>
+    keys() const noexcept
+    {
+        std::vector<std::string> r;
+        for (auto p : _map)
```

Comment:
auto makes code harder to read. If the type is not convoluted it is better to use that instead of auto.

Reply:
Good point—sometimes I get lazy with `auto` :) Fixed in [ca4ce00](https://github.com/wiredtiger/wiredtiger/pull/10251/commits/ca4ce0065a9940d3d15c15c36ee8877c7bedce9c).